PressureAdapter.java

```diff
@@ -0,0 +1,34 @@
+package sensor;
+
+import java.awt.Color;
+
+//Implements SensorInterface using PressureSensor methods
+public class PressureAdapter implements SensorInterface {
+	PressureSensor ps = new PressureSensor();
+	
+	@Override
+	public String getName() {
+		return ps.getSensorName();
+	}
+
+	@Override
+	public String getReport() {
+		return ps.getReport();
+	}
+
+	@Override
+	public double getValue() {
+		return ps.readValue();
+	}
+
+	//returns color based on report string
+	public Color getColor() {
+		String report = getReport();
```

Comment:
there is a lot of room for error when comparing strings. You'd be safer adding a style conversion incase the string passed from her PressureSensor class wasn't in all caps. _String report = getReport().toLowerCase();_